lib/bloc/backup/backup_bloc.dart

```diff
@@ -337,13 +475,48 @@ class BackupBloc {
 
       _breezLib
           .restore(request.snapshot.nodeID, request.encryptionKey.key)
-          .then((_) => _restoreFinishedController.add(true))
-          .catchError(_restoreFinishedController.addError);
+          .then((_) => _restoreAppData()
+              .then((value) => _restoreFinishedController.add(true))
+              .catchError(_restoreFinishedController.addError));
     });
   }
 
+  _restoreAppData() async {
+    try {
+      await _restorePosDB();
+      await _restorePodcastsDB();
+    } on Exception catch (exception) {
+      throw exception;
+    }
+  }
+
+  Future<void> _restorePosDB() async {
+    final backupPosDbPath =
+        _backupAppDataDirPath + Platform.pathSeparator + 'product-catalog.db';
+    final posDbPath = await databaseFactory.getDatabasesPath() +
+        Platform.pathSeparator +
+        'product-catalog.db';
+    if (await File(backupPosDbPath).exists()) {
+      await File(backupPosDbPath).copy(posDbPath).catchError((err) {
```

Comment:
Maybe it is better to delete after restore? So in case there is an error and the user decides to restore from a different backup he won't end with the old app data?